docs/platform-services/automation-service/automation-service-audit-logging.md

```diff
@@ -0,0 +1,136 @@
+---
+id: automation-service-audit-logging
+title: Audit Logging for the Automation Service
+sidebar_label: Audit Logging
+description: Learn how to search the Audit Event Index for Automation Service log events.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The [Audit Event Index](/docs/manage/security/audit-indexes/audit-event-index/) provides event logs in JSON format on your account activity allowing you to monitor and audit changes. By default, the Audit Event Index is enabled for the Automation Service.
+
+:::note
+Audit logging for the Automation Service uses the same logging as Cloud SOAR, since the Automation Service is based on core functionality in Cloud SOAR. See [Cloud SOAR Audit Logging](/docs/cloud-soar/audit-event-index/).
+:::
+
+## Search the Audit Event Index
+
+Searching the Audit Event Index is the same as running a normal search against your ingested data. You specify the `_index` metadata field with one of these values: 
+
+* `sumologic_audit_events`. This index contains user action events, which are events that were triggered by a user action, either from the UI or an API.
+* `sumologic_system_events`. This index contains system action events, which are events that were triggered by Sumo Logic. For example, this index contains Automation Actions start events, rules triggered, and so on.
+
+## Automation Service audited events
+
+The Audit Event Index has detailed JSON logs for the following features. 
```

Comment:
Double-checking that this sentence belongs here, since the section isn't showing JSON logs

Reply:
You're right. I'll remove it.